packages/controllers/src/claim.rs

```diff
@@ -90,6 +91,28 @@ impl<'a> Claims<'a> {
         let claims = self.0.may_load(deps.storage, address)?.unwrap_or_default();
         Ok(ClaimsResponse { claims })
     }
+
+    pub fn range_claims(
+        &self,
+        deps: Deps,
+        min: Option<Bound>,
```

Comment:
This is different than most list queries. I assume this would be used by such a query? Or is this for "internal use only"?

I would either:

* use `start_after` and `limit` (assigning default/max to limit if not set) like all `list_*` queries.
* or just expose `self.0.range()` with the identical arguments, not trying to add limits, using `&dyn Storage` not `Deps`, return Iterator, etc

Those are the 2 existing APIs people use. I see no reason to invent a 3rd API for the same approach (the other 2 are internal vs external, just pick one)

Actually, I think the simplest implementation for the 2nd approach is to expose the whole map read-only like, by implementing Deref here (so I can do `CLAIMS.range(...)` and it will auto-call the Map, but it doesn't let me modify the Map (only get a &Map)